lib/src/view/play/challenge_screen.dart

```diff
@@ -332,15 +382,23 @@ class _ChallengeBodyState extends ConsumerState<_ChallengeBody> {
                     child: FatButton(
                       semanticsLabel: context.l10n.challengeChallengeToPlay,
                       onPressed: timeControl == ChallengeTimeControlType.clock
-                          ? isValidTimeControl
+                          ? isValidTimeControl &&
+                                  ((textInput != '' && textInput != null) ||
```

Comment:
I think you can use `textInput.isNotEmpty` instead of comparing with empty string.